config.go

```diff
@@ -288,15 +305,14 @@ func DefaultConfig() *Config {
 		Format:              "flat_json",
 		FormatRollup:        "",
 		Compression:         "none",
-		Sinks:               "stdout",
+		Sinks:               []string{"stdout"},
```

Comment:
Will this break current deployments with the comma separated string?

Reply:
disregard -- i didn't see it in the adapter

Reply:
yeah -- I think the split in cmd/ktransate should take care of backwards compatibility.